src/core/navigation/navigation.view.js

```diff
@@ -48,11 +48,6 @@ export class NavigationView {
 						switch (this.model.selection().unit) {
```

Comment:
u can remove this too) its useless

Reply:
i didn't get what you want me to delete. 
line 5: import { Td } from '../dom/td'; or the whole "IF" statement from line 44?
both are useless, i guess.

Reply:
what about line 5?